types/react-native-canvas/index.d.ts

```diff
@@ -165,6 +136,7 @@ export default class Canvas extends React.Component<CanvasProps> {
 }
 
 export class Image {
+    constructor(canvas: Canvas, height: number, width: number);
```

Comment:
This is redundant with the already existing constructor declaration, isn't it?

Reply:
You are right! This seems to be a mistake when I moved the code around. Thanks for spotting!